### PR TITLE
Add Validation and Refactor the Code

### LINKED ISSUE #1: Refactor existing code
Hey devansh, I would like to refactor your some parts of the code. Can you provide some kind of JSON data for testing purpose.

### PR DESCRIPTION
**Pull request by varshil-shah**
Add strict validation in event-model file. Converted all normal functions to arrow functions.

### CODE REVIEW

**Comment by devansh016** [comment]
why converted to arrow function?

**Comment by varshil-shah** [comment]
It's javascript modern syntax of writing functions

File: models/event-model.js [hunk 1]
```diff
@@ -2,10 +2,29 @@ const mongoose = require("mongoose");
...
+            },
+            message: 'An event title should contain only alphabets',
+        },
+    },
+    description: { 
+        type: String,
+        trim: true,
+        required: [true, 'Please provide a description for an event'],
+    },
+    date: { type: Date, required: [true, 'Please provide date for an event'] },
```
Comment (devansh016) [inline models/event-model.js]: fix indentation here

Comment (varshil-shah) [reply models/event-model.js]: I'll fix it by tomorrow

**Comment by varshil-shah** [comment]
I've fixed the indentation, please check the same

**Pull request merged**